scripts/pi-hole/js/groups-lists.js

```diff
@@ -400,7 +400,7 @@ function initTable() {
             ids.push($(this).attr("data-id"));
```

Comment:
This is missing for Lists:

```suggestion
            ids.push({ item: $(this).attr("data-id") });
```